build.package.xml

```diff
@@ -67,6 +67,10 @@
             <include name="*.*" />
           </fileset>
         </delete>
+        <composer command="install" composer="${composer.bin}">
```

Comment:
I am not confident with these lines, it's kind of double work as the make file manage to do it.